### DISCUSSION TITLE
Error in Source Connector "YouTube Analytics: "The YouTube account is not valid."

### DISCUSSION THREAD

**Discussion by smdraeger** [airbytehq/airbyte#44582] [Connector Questions]
Hi, we have a problem with the airbyte open source connector ‘youtube analytics’ (https://docs.airbyte.com/integrations/sources/youtube-analytics).

We followed the instructions in the documentation exactly and receive the error message ‘The YouTube account is not valid. Please make sure you're trying to use the active YouTube Account connected to your Google Account.’ see screenshot below ![image](https://github.com/user-attachments/assets/94945d4d-38e9-4dd0-9cb1-1718dfb6dc4c)

We made sure that within YouTube Channel the Google Account has ‘Manager’ rights - the same google account which have generated the refresh_token. The YouTube account is active. Scope within Google Api configuration has been double-checked.

Do you have any ideas how to fix this?
Thanks

**Comment by smdraeger** [comment]
Hi there, any ideas here? Thanks

**Comment by nataliekwong** [comment]
Are you possibly using a brand account? We've seen this issue before, but never figured out what the reason for this was, as some users have no issues authenticating and others run into this blocker. 

If you are, I found the following articles that could help: 
- https://stackoverflow.com/questions/76677463/youtube-data-api-connecting-to-a-brand-account
- https://support.google.com/youtube/answer/4628007?sjid=4886304586147156025-NC

**Comment by therohc** [comment]
I had the same error and I think I was able to narrow down the causes for it, or at least what worked for me, just in case anyone else is having similar issues:

 * Make sure the account for which you are fetching data for is a "Brand Account" (https://support.google.com/youtube/answer/7001996) if you are not the owner of the channel. 
 * **The integration does not create/fails to create reporting  jobs if none exist.**  If I first create at least one report job, after doing that the integration was able to create the jobs for the rest of reports and the error goes away.

Hope this helps.

**Reply by ypubalan** [reply]
Hi @therohc the brand account has to be set in my google account settings or youtube account settings. And do I need an invitation to make my google account a brand account. And also, is editor role sufficient in youtube studio or do I need manager role for this.
